lib/server.js

```diff
@@ -1,20 +1,24 @@
 var Botkit = require('botkit');
+require("dotenv").config();
+var config = require('../config').config;
 
 var controller = Botkit.slackbot({
   debug: false
 });
 
+
 var bot = controller.spawn({
-  token: process.env.PROJECT_GREETER_SLACK_BOT_TOKEN,
+  token: config[process.env.ENV].PROJECT_GREETER_SLACK_BOT_TOKEN,
```

Comment:
also, I think this would be better if it was like `process.env.NODE_ENV` it's not a requirement, it's a pattern I have seen used a lot.

Reply:
Thanks you are right.